README.md

```diff
@@ -142,28 +141,28 @@ Once Prisma Client was generated, you can import in your code and send queries t
 You can import and instantiate Prisma Client as follows:
 
 ```ts
-import { PrismaClient } from '@prisma/client'
+import { PrismaClient } from "@prisma/client";
```

Comment:
Why reformat?

Reply:
It just happened automatically 😅 should I revert?